packages/ui/src/components/Table/BaseTable.tsx

```diff
@@ -62,6 +63,12 @@ export interface BaseTableProps<T extends GenericRowStructure> {
     rows: Row<T>[];
     rowActions?: RowAction<T>[];
     ellipsisIcon?: ReactNode;
+    //
+    rowsPerPageOptions?: number[];
```

Comment:
if you want to differentiate those props from the rest, you can nest them under "sortingProps" or something similar

Reply:
That's left by mistake, i'll remove them